package.json

```diff
@@ -14,8 +16,9 @@
     "solc": "^0.5"
```

Comment:
what about this solc dep?

Reply:
package.json deps don't matter for purescript -- it's worth noting that this library allows you to invoke /any/ version of the solidity compiler (see the FFI in Compiler.js) -- the API was just stabilized in 0.5+

Reply:
(Releases.purs fetches/loads whatever version you want)